BraveShared/Analytics/UserReferralProgram.swift

```diff
@@ -16,6 +16,7 @@ public class UserReferralProgram {
     public static let shared = UserReferralProgram()
     
     private static let apiKeyPlistKey = "API_KEY"
+    private static let clipboardPrefix = "F83AB73F-9852-4F01-ABA8-7830B8825993"
```

Comment:
Would we maybe want to place this at build time/in PLIST or maybe even pull from servers? Or do we expect this to not change ever

Reply:
Very unlikely to change, as would require changes to our partner landing pages. I did verify that no sec concerns with this. If you think better elsewhere I can remove. It could just as well be something like `Brave_Referral_Partnership`, was requested to make it something super random though.